src/customer.js

```diff
@@ -26,9 +27,45 @@ let roomData = fetch('https://fe-apps.herokuapp.com/api/v1/overlook/1904/rooms/r
 
 
 Promise.all([bookingData, roomData]).then((requiredData) => {
-  let userID = parseInt(localStorage.getItem('userID'))
-  const rooms = requiredData[1];
-  const booked = requiredData[0];
-  customer.myBookings(booked, userID).forEach(room => {$('.bookings').append(`<p class="rooms">Room: ${room.roomNumber} ${room.date}</p>`)})
-  $('.spending').html(customer.totalSpend(rooms, customer.myBookings(booked, userID)))
+  userID = parseInt(localStorage.getItem('userID'))
+  customer.rooms = requiredData[1];
+  customer.booked = requiredData[0];
```

Comment:
Oh!  I see where you are passing this info into customer now.   

I would consider refactoring this so that the customer is being instantiated after the promise.all is resolved.   
Then you could pass these two datasets in as parameters.

`let customer = new Customer(requiredData[1], requiredData[0])`

One of the concepts I keep in mind for SRP is that any properties on a class should only be able to be modified by methods within that class.   I think modifying these to properties in your customer.js file may violate that.